src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TaxTableRow.java

```diff
@@ -223,6 +224,63 @@ final TaxTableField getAliasedField(String alias) {
+        final TaxTableField financialObjectCode;
+        
+        // Fields from AP_PMT_RQST_ITM_T 
+        final TaxTableField purapDocumentIdentifier;
+        final TaxTableField itemIdentifier;
+
+        // Fields from AP_PMT_RQST_T (PaymentRequestDocument)
+        final TaxTableField preqDocumentNumber;
+        final TaxTableField preqPurapDocumentIdentifier;
+        final TaxTableField paymentMethodCode;
+        final TaxTableField taxClassificationCode;
+        final TaxTableField preqVendorHeaderGeneratedIdentifier;
+        final TaxTableField preqVendorDetailAssignedIdentifier;
+        final TaxTableField preqVendorName;
+        final TaxTableField vendorLine1Address;
+        final TaxTableField vendorCountryCode;
+        
+        // Fields from SH_UNIV_DATE_T (UniversityDate)
+        final TaxTableField universityDate;
+
+        
+        PRNCSourceRow(String rowId, Map<String,TaxTableField> fields, List<String> tables, Map<String,TaxTableField> aliasedFields, Integer insertOffset) {
+            super(rowId, fields, tables, aliasedFields, insertOffset);
+            
+            this.accountIdentifier = getAliasedField(CommonPRNCSourceFieldNames.ACCOUNTING_LINE_IDENTIFIER);
+            this.accountItemIdentifier =  getAliasedField(CommonPRNCSourceFieldNames.ACCOUNTING_LINE_ITEM_IDENTIFIER);
+            this.amount =  getAliasedField(CommonPRNCSourceFieldNames.AMOUNT);
+            this.chartOfAccountsCode = getAliasedField(CommonPRNCSourceFieldNames.CHART_OF_ACCOUNTS_CODE);
+            this.accountNumber = getAliasedField(CommonPRNCSourceFieldNames.ACCOUNT_NUMBER);
+            this.financialObjectCode = getAliasedField(CommonPRNCSourceFieldNames.FIN_OBJECT_CODE);
+            this.purapDocumentIdentifier = getAliasedField(CommonPRNCSourceFieldNames.PURAP_DOC_IDENTIFIER);
+            this.itemIdentifier = getAliasedField(CommonPRNCSourceFieldNames.ITEM_IDENTIFIER);           
+            this.preqDocumentNumber = getAliasedField(CommonPRNCSourceFieldNames.PREQ_DOCUMENT_NUMBER);
+            this.preqPurapDocumentIdentifier = getAliasedField(CommonPRNCSourceFieldNames.PREQ_PURAP_DOC_IDENTIFIER);
+            this.paymentMethodCode = getAliasedField(CommonPRNCSourceFieldNames.DOCUMENT_PREQ_PAYMENT_METHOD_CODE); 
+            this.taxClassificationCode = getAliasedField(CommonPRNCSourceFieldNames.DOCUMENT_PREQ_TAX_CLASSIFICATION_CODE);
+            this.preqVendorHeaderGeneratedIdentifier = getAliasedField(CommonPRNCSourceFieldNames.PREQ_VENDOR_HEADER_GENERATED_ID);
+            this.preqVendorDetailAssignedIdentifier = getAliasedField(CommonPRNCSourceFieldNames.PREQ_VENDOR_DETAIL_ASSIGNED_ID);
+            this.preqVendorName = getAliasedField(CommonPRNCSourceFieldNames.PREQ_VENDOR_NAME);
+            this.vendorLine1Address = getAliasedField(CommonPRNCSourceFieldNames.PREQ_VENDOR_LINE1_ADDRESS);
+            this.vendorCountryCode = getAliasedField(CommonPRNCSourceFieldNames.PREQ_VENDOR_COUNTRY_CODE);;

```

Comment:
Unnecessary double semicolon, but I don't think it's worth fixing unless other issues come up that would require further changes.